### DISCUSSION TITLE
ERROR: The Procedure entry point setDefaultDllDirectories could not be located........

### DISCUSSION THREAD

**Discussion by ghost** [atom/atom#22173] [Q&A]
![tmp](https://user-images.githubusercontent.com/68454783/114293242-20cb9080-9ab2-11eb-8a89-cf38fa628235.png)

My Windows Specifications:

![tmp](https://user-images.githubusercontent.com/68454783/114293279-6be5a380-9ab2-11eb-9f5f-4907e12445ce.png)

**Comment by lamdalili** [comment]
Same thing .. 
J'espere qu'il s'agit d un simple erreur et pas un virus introduite dans le programme d’installation 

 W7
 
![erreur](https://user-images.githubusercontent.com/40634105/114316481-0c17e880-9b04-11eb-9699-98561a9b3dca.png)
